Marlin/Configuration.h

```diff
@@ -1361,6 +1361,7 @@
     #define MESH_TEST_HOTEND_TEMP  205    // (°C) Default nozzle temperature for the G26 Mesh Validation Tool.
     #define MESH_TEST_BED_TEMP      60    // (°C) Default bed temperature for the G26 Mesh Validation Tool.
     #define G26_XY_FEEDRATE         20    // (mm/s) Feedrate for XY Moves for the G26 Mesh Validation Tool.
+    #define G26_XY_FEEDRATE_TRAVEL    150   // (mm/s) Feedrate for XY Moves without extrusion for the G26 Mesh Validation Tool
```

Comment:
@ScrewThisBanana Could you align it, like the values above?

Reply:
Ofc

Reply:
https://github.com/MarlinFirmware/Marlin/pull/20729/commits/aaa873e4eb6a13d09f16a118d0da551a3f558a00